README.md

```diff
@@ -1,13 +1,229 @@
+
+<img src="https://avatars.githubusercontent.com/u/22056578?v=4" width="150px" />
+
+#### Nico Rithner
+
+Github: [github.com/nicorithner](https://github.com/nicorithner) 
+
+LinkedIn: [linkedin.com/in/nicorithner/](https://www.linkedin.com/in/nicorithner/)
+
+</div>
+
+## Goals
+
+### Project First Iteration Goals
+
+- A crud a API with Users, Menus, and Recipes
+- Recipes are generated by an external API - [spoonacular api](https://spoonacular.com/food-api) but transformed into a new recipe format before sending a JSON response through our API.
+- APIs include tests
+- APIs are documented using swagger
+
+### Extension Goals for Future Iterations
+
+- A User can create a menu indicating preferred ingredients
+- A User can update a menu recipes individually without having to regenerate the whole batch.
+- Implement User authentication
+
+## Stack
+
+- NodejS
+- ExpressJS
+- TypeScript
+- TypeOrm
+- PosgreSQL
+
+## Install Locally
+
+To run locally please first make sure to already have installed [Node](https://nodejs.org/en) and [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+- Clone this repo locally `git clone git@github.com:nicorithner/chinese-menu-generator-be.git`
+
+To run the api:
+
+- Run `npm install`
+- Run `npm run build`
+- Run `npm run dev`
+
+It will run at `http://localhost:8000/`
+
+To test:
+
+- Run `npm run test`
+
+If this fails run tests individually:
+
+- Run `npm run test __tests__/routes/users.test.ts`
+- Run `npm run test __tests__/routes/menus.test.ts`
+- Run `npm run test __tests__/routes/recipes.test.ts`
+
+## Schema
 
 <div align="center">
 <img src="assets/ERD.png" width="500"/>
 </div>
 
-## Documentation - Swagger
+### User
+
+  - Id
+  - firstName
+  - lastName
+
+A user has many menus
+
+### Menu
+
+  - Id
+  - name
+  - user_id
+  
+A menu belongs to a user
+
+### Recipe
+
+  - Id
+  - category
+  - title
+  - summary
+  - steps
+  - ingredients
+
+A recipe belongs to a menu
+
+## Swagger
 
 To access API documentation please navigate to
 
-`/api-docs`
+`http://localhost:8000/api-docs`
+
+## Endpoints
+
+### Postman
+
+Visit postman to checkout the collection of enpoints and try it locally.
+
+
+### User Endpoints
+
+##### GET "/users"
+
+    Finds all users
+
+##### GET "/users:id"
+
+    Finds an individual user by id
+
+##### POST "/users"
+
+    Creates a new user
+
+##### PUT "/users/:id"
+
+    Updates an individual user by id
+
+##### DELETE "/users/:id"
+
+    Deletes an individual user by id
+
+##### GET "/users/:id/menus"
+
+    Lists a user by id menus
+
+<hr/>
+
+### Menu Enpoints
+
+##### GET "/menus"
+
+    Finds all menus
+
+##### GET "/menus:id"
+
+    Finds an individual menu by id
+
+##### POST "/menus"
+
+    Creates a new menu
+
+##### PUT "/menus/:id"
+
+    Updates an individual menu by id
+
+##### DELETE "/menus/:id"
+
+    Deletes an individual menu by id
+
+##### GET "/menus/:id/menus"
```

Comment:
This one should be: 
DELETE "/menus/:id/recipeId"
Deletes the the menu recipe relationship in the menu recipe join table

Reply:
✅